src/charts/TileMap.js

```diff
@@ -16,20 +55,63 @@ function TileMap( props ) {
     title: {
       text: props.title
     },
+    chart: {
+      marginTop: 150
+    },
     description: props.description,
     credits: false,
     legend: {
       enabled: false
     },
+    // tooltip: {
```

Comment:
Having tooltips is nice but I couldn't get them working so I left that code there for y'all. If you have time, revisit them.